cmd/bridgec/internal/client.go

```diff
@@ -0,0 +1,119 @@
+package internal
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"runtime"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/threefoldtech/rivine/pkg/api"
+	"github.com/threefoldtech/rivine/pkg/cli"
+)
+
+// CommandLineClient extend for ERC20 commands
+type CommandLineClient struct {
+	*api.HTTPClient
+
+	RootCmd  *cobra.Command
+	ERC20Cmd *cobra.Command
+}
+
+// NewCommandLineClient creates a new CLI client, which can be run as it is,
+// or be extended/modified to fit your needs.
+func NewCommandLineClient(address, name, userAgent string) (*CommandLineClient, error) {
+	if address == "" {
+		address = "http://localhost:23111"
+	}
+	if name == "" {
+		name = "R?v?ne"
+	}
+	client := new(CommandLineClient)
+	client.HTTPClient = &api.HTTPClient{
+		RootURL:   address,
+		UserAgent: userAgent,
+	}
+
+	client.RootCmd = &cobra.Command{
+		Use:   os.Args[0],
+		Short: fmt.Sprintf("%s Client", strings.Title(name)),
+		Long:  fmt.Sprintf("%s Client", strings.Title(name)),
+		Run: Wrap(func() {
+			fmt.Println("Bride Client v1.0.0")
```

Comment:
Let's not use a custom version. Use the same version number as the other code. Look in tfchainc and tfchaind where they get the version from and use the same one. All binaries get released at the same time, so having a different version does not just give more work, it also serves no point.